### PR TITLE
Feature/tiny 4717

### PR DESCRIPTION
**Pull request by tinydylan**

### CODE REVIEW

File: modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestStore.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ interface TestStore {
   sAssertSortedEq: (label: string, expected: any[]) => Step<any, any>;
 }
 
-const TestStore = (): TestStore => {
+export const create = (): TestStore => {
```
Comment (lnewson) [inline modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestStore.ts]: If you name this `TestStore` as well then there'd be less you need to change and the types will merge, so you wouldn't need to do the `type TestStore = TestStore.TestStore;` everywhere?

Comment (lnewson) [reply modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestStore.ts]: Doing the above would also mean this isn't an API breaking change, so given we don't need this to be an API breaking change, I'd suggest changing this to:
```
export const TestStore = (): TestStore => {
```

Comment (tinydylan) [reply modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestStore.ts]: Does this look better?

Comment (lnewson) [reply modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestStore.ts]: Looks good, thanks! :+1:

**Comment by tinydylan** [comment]
> Did this end up making any difference to the bundle size?

Negligible.

**Pull request merged**